vmdb/app/models/vm_redhat.rb

```diff
@@ -37,4 +37,12 @@ def self.calculate_power_state(raw_power_state)
     else             super
     end
   end
+
+  def supports_migrate?
+    false
+  end
+
+  def supports_reconfigure?
+    false
+  end
```

Comment:
That's a good point, and is probably a deficiency in the validate_xx methods that we should correct eventually.  @h-kataria Do you only have vm_ids, or do you also have class types in the UI?  There might be some shorcuts / performance improvements if you kept class/id pairs.